### DISCUSSION TITLE
1.5.1.0

### DISCUSSION THREAD

**Discussion by saniho** [saniho/apiEnedis#141] [General]
je pense qu'on peut releaser la version en 1.5.1.0
ok pour toi ?
Il reste juste deux soucis : 
* le merge sur /main
* le archive qui n'est pas au bon endroit( resté à l'origine )

**Comment by mdeweerd** [comment]
Pour moi cela fonctionne comme d'habitude: des fois j'ai des données, des fois non.

Il faut probablement aussi revoir la répétition des requêtes quand la dernière n'a pas vraiment abouti.

Je m'occupes de préparer un merge dans le main.
